WindowsRequirements.json

```diff
@@ -3,7 +3,8 @@
     "brotli",
     "libressl",
     "nghttp2",
-    "curl[ca-bundle,ssl,ipv6]",
+    "c-ares",
+    "curl[ares,ca-bundle,ssl,ipv6]",

```

Comment:
I think we should probably not yet default curl to using ares until the network guys are a bit more ready

Reply:
Fixed